src/pages/o-meu-codigo-postal.jsx

```diff
@@ -11,9 +11,11 @@ import "@/styles/main.module.css";
 const getParamFromPathname = pathname => pathname.split("/").pop();
 
 const checkIfImageExists = (imageUrl, bad) => {
-  const img = new Image();
-  img.onerror = bad;
-  img.src = imageUrl;
+  if (typeof window !== "undefined") {
```

Comment:
I didn't though this will work. On merge, please discard my version and use this fix, please

Reply:
Done!